ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__vsphere-4.18-nightly-x86.yaml

```diff
@@ -97,6 +97,58 @@ tests:
     - chain: openshift-qe-node-density-heavy
     workflow: openshift-qe-installer-vsphere-ipi-ovn
   timeout: 5h0m0s
+- always_run: false
+  as: vcenter-1-control-plane-3nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: vsphere-elastic
+    env:
+      DEFAULT_NETWORK_TYPE: public-ipv6
+      POOLS: vcenter-1.devqe.ibmc.devcluster.openshift.com-devqedatacenter-1-devqecluster-1
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-vsphere-ipi-multi-vcenter
+  timeout: 5h0m0s
+- always_run: false
+  as: vcenter-1-control-plane-24nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: vsphere-elastic
+    env:
+      DEFAULT_NETWORK_TYPE: public-ipv6
+      POOLS: vcenter-1.devqe.ibmc.devcluster.openshift.com-devqedatacenter-1-devqecluster-1
+      WORKER_REPLICA_COUNT: "24"
+    test:
+    - chain: workers-scale
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-vsphere-ipi-multi-vcenter
+  timeout: 5h0m0s
+- always_run: false
+  as: vcenter-1-ipv4ipv6dual-control-plane-3nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: vsphere-elastic
+    env:
+      DEFAULT_NETWORK_TYPE: public-ipv6
+      POOLS: vcenter-1.devqe.ibmc.devcluster.openshift.com-devqedatacenter-1-devqecluster-1
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-vsphere-ipi-dualstack
+  timeout: 5h0m0s
+- as: vcenter-1-ipv4ipv6dual-control-plane-24nodes
+  cron: 0 7 2,9,16,23 * *
```

Comment:
be sure to define the number of cores and memory in GB you'll need for each job.  eventually, we may start enforcing resource leasing and constraints in devqe to ensure all users of the environment are able to run their tests.

https://github.com/openshift/release/blob/99e85f00df4aceed6f7d18bfc6c937f40ac91667/ci-operator/step-registry/ipi/conf/vsphere/check/vcm/ipi-conf-vsphere-check-vcm-ref.yaml#L19-L25

Reply:
@rvanderp3 Are you sure to have such big size as default? For our team 4 CPU cores 16G memory is enough as a default.

Reply:
> @rvanderp3 Are you sure to have such big size as default? For our team 4 CPU cores 16G memory is enough as a default.

6 nodes x 4 cpus = 24
6 nodes x 16gb = 96

its the total cores and ram required